modules/router-store/spec/utils.ts

```diff
@@ -54,6 +54,13 @@ export function createTestModule(
         provide: 'CanLoadNext',
         useValue: opts.canLoad || (() => true),
       },
+      {
+        provide: Console,
```

Comment:
Overriding console logs, otherwise the console will be bloated with warnings that the `router.navigate` runs outside Angular zone. https://github.com/angular/angular/pull/24959